readme.md

```diff
@@ -760,6 +760,13 @@ Default: `Infinity`
 
 The maximum amount of items that should be emitted.
 
+###### pagination.requestLimit
+
+Type: `number`\
+Default: `100`
+
+The maximum amount of request that should be triggered.
```

Comment:
The text should include the intended use-case for this option.

Reply:
I just updated the paragraph with an example.

Reply:
It should be noted that retry requests does not count.

Reply:
Just added that note as well.